module/Public/ConvertTo-SplatExpression.ps1

```diff
@@ -11,6 +11,9 @@ function ConvertTo-SplatExpression {
     param(
         [System.Management.Automation.Language.Ast]
         $Ast
+        
```

Comment:
`$Ast` needs a comma here to prevent a parse exception

Reply:
Doh.. fixed!